odoo/tests/test_parse_inline_template.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import BaseCase
+from odoo.tools import parse_inline_template
+
+class TestParseInlineTemplate(BaseCase):
```

Comment:
Still a bit light + would add tests for the block-text inline rendering method I propose, if you add it (wink wink).